docs/internals/howto-release-django.txt

```diff
@@ -188,8 +188,11 @@ A week before a security release
 A few days before any release
 -----------------------------
 
-#. As the release approaches, watch Trac to make sure no release blockers
-   are left for the upcoming release.
+#. As the release approaches, watch Trac to make sure no release blockers are
+   left for the upcoming release. Under exceptional circumstances, such as to
+   meet a pre-determined security release date, a release could still go ahead
+   with an open release blocker. The releaser is trusted with the decision to
```

Comment:
Just to be explicit, I wonder if this sentence just needs a, `For non-security releases, ...` 

I'm not sure it does.

Reply:
Like: `The releaser is trusted with the decision to release with an open release blocker and to postpone the release date of a non-security release if required.`?

Reply:
Yeah, better. Thanks. 🎁